pages/linux/autorecon.md

```diff
@@ -0,0 +1,21 @@
+# autorecon
+
+> A multi-threaded network reconnaissance tool which performs automated enumeration of services.
+> Note: dependency packages need to be installed for `autorecon` to function properly.
```

Comment:
This is the case for all programs. Should we list the dependencies explicitly here?

Reply:
Removed the line.